utils/config.go

```diff
@@ -278,16 +278,13 @@ func EnsureConfigFile(fileName string) (string, error) {
 	return "", fmt.Errorf("no config file found")
 }
 
-// LoadConfig will try to search around for the corresponding config file.  It will search
-// /tmp/fileName then attempt ./config/fileName, then ../config/fileName and last it will look at
-// fileName.
-func LoadConfig(fileName string) (config *model.Config, configPath string, appErr *model.AppError) {
+func loadConfigFile(fileName string) (config *model.Config, configPath string, appErr *model.AppError) {
```

Comment:
I actually just realized a potential problem here. What happens if the user specifies the path to their config.json using an environment variable and we try to load the timezones.json? This affects any GitLab Omnibus installs in particular.

It might be a bit hokey, but we might need to add a config setting for the path to the timezones file so that it can be loaded properly. Something like `TimezoneSettings.AvailableTimezonesPath` that, if empty, defaults to doing what it does currently?

Reply:
> Are these going to get written back to the config.json when that gets saved?

Normally when is it saved back to the config.json file?
I noticed admin can save the config, so I can add some logic to filter out `SupportedTimezones` in order to not update `config.json` with those values


> It might be a bit hokey, but we might need to add a config setting for the path to the timezones file so that it can be loaded properly. Something like TimezoneSettings.AvailableTimezonesPath that, if empty, defaults to doing what it does currently?

I can see this being useful. I can add that in 👍